win32/tests/file_win32_ut/mock_file.h

```diff
@@ -0,0 +1,36 @@
+#include "windows.h"
+#include "umock_c/umock_c_prod.h"
+#include "azure_c_pal/execution_engine.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+MOCKABLE_FUNCTION(, HANDLE, mock_CreateFileA, LPCSTR, lpFileName, DWORD, dwDesiredAccess, DWORD, dwShareMode, LPSECURITY_ATTRIBUTES, lpSecurityAttributes, DWORD, dwCreationDisposition, DWORD, dwFlagsansAttributes, HANDLE, hTemplateFile);
+MOCKABLE_FUNCTION(, BOOL, mock_SetFileCompletionNotificationModes, HANDLE, FileHandle, UCHAR, Flags);
+MOCKABLE_FUNCTION(, void, mock_InitializeThreadpoolEnvironment, PTP_CALLBACK_ENVIRON, pcbe);
+MOCKABLE_FUNCTION(, void, mock_SetThreadpoolCallbackPool, PTP_CALLBACK_ENVIRON, pcbe, PTP_POOL, ptpp);
+MOCKABLE_FUNCTION(, PTP_CLEANUP_GROUP, mock_CreateThreadpoolCleanupGroup);
+MOCKABLE_FUNCTION(, void, mock_SetThreadpoolCallbackCleanupGroup, PTP_CALLBACK_ENVIRON, pcbe, PTP_CLEANUP_GROUP, ptpcg, PTP_CLEANUP_GROUP_CANCEL_CALLBACK, pfng);
+MOCKABLE_FUNCTION(, PTP_IO, mock_CreateThreadpoolIo, HANDLE, fl, PTP_WIN32_IO_CALLBACK, pfnio, PVOID, pv, PTP_CALLBACK_ENVIRON, pcbe);
+MOCKABLE_FUNCTION(, void, mock_CloseThreadpoolIo, PTP_IO, pio);
+
+
+

```

Comment:
nit: collapse new lines

Reply:
done

---
In reply to: [456716313](https://github.com/Azure/azure-c-pal/pull/10#discussion_r456716313) [](ancestors = 456716313)